src/main/java/WriteFile.java

```diff
@@ -0,0 +1,39 @@
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class WriteFile {
+
+    public static void writeToFile(String filePath, ArrayList<Task> listOfTasks) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+
+        for(int i = 0; i<listOfTasks.size();++i){
```

Comment:
Try to use spaces here in this statement. For example:
for ( int i = 0 ; i < listOfTasks.size(); ++i ) {
......
}